test/InterchainTokenFactory.js

```diff
@@ -50,6 +51,40 @@ describe('InterchainTokenFactory', () => {
         });
     });
 
+    describe('Upgrade', async () => {
```

Comment:
Should we add an upgrade test to `InterchainTokenService` since it is also 'Upgradable'?

Reply:
it has a dedicated upgrade test in `InterchainTokenServiceUpgradeFlow.js`

Reply:
Might move the factory test there in the future, not sure